controllers/skills.js

```diff
@@ -0,0 +1,70 @@
+const skills = require("../models/skills");
+const logger = require("../utils/logger");
+
+/**
+ * Award a skill
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - User object
+ * @param res {Object} - Express response object
+ */
+async function awardSkill(req, res) {
+  try {
+    const { username } = req.params;
+    const addedData = await skills.awardSkill(req.body, username);
```

Comment:
Add validation for data being passed to model

Reply:
validation for?